dask/dataframe/core.py

```diff
@@ -604,6 +606,7 @@ def rolling(self, window, min_periods=None, win_type=None):
 
         return Rolling(self, {'window': window,
                               'min_periods': min_periods,
+                              'axis': axis,
                               'win_type': win_type})
```

Comment:
I'm not entirely sure what it buys you other than having to type the same four or five fields several times and keep them synced up.